docs/workflow.md

```diff
@@ -31,3 +31,16 @@ To simulate Axe for Android (running on port 9051) for local testing of the app,
 ```sh
 yarn with:mock-axe-android start:electron:dev
 ```
+
+## Pull Request title tagging convention (Labelling commit messages)
+
+For every Pull Request, we prefer to add a tag in the PR title. This helps us in maintaining a good commit history as well as better documentation
+when creating Release notes.
+The convention that we follow is inspired from [SemVer](https://semver.org/) convention and is following:
+
+| Tag        | What it conveys                                                                                                                             |
+| ---------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| feat.(...) | a commit of the type feat. introduces a new feature in master or means completion of task towards a feature                                 |
```

Comment:
Looking at the PR history, we haven't included a period in `feat`; should it be removed here?

Reply:
I guess to maintain consistency, we should. I will update.